extensions/amp-bind/0.1/test/test-bind-expression.js

```diff
@@ -268,6 +268,13 @@ describe('BindExpression', () => {
     }).to.throw(unsupportedFunctionError);
   });
 
+  it('should support encodeURI and encodeURIComponent', () => {
+    expect(evaluate('encodeURI("http://www.google.com/s p a c e.html")'))
+        .to.equal('http://www.google.com/s%20p%20a%20c%20e.html');
+    expect(evaluate('encodeURIComponent("hello world")'))
+        .to.equal('hello%20world');
```

Comment:
I'm not sure that's necessary. We don't need to test the correctness of the functions, just that using them in bind expressions works.